products/ocp4/profiles/cis.profile

```diff
@@ -118,16 +118,10 @@ selections:
     - api_server_tls_cipher_suites
   #### 1.3 Controller Manager
   # 1.3.1 Ensure that garbage collection is configured as appropriate
-    - kubelet_eviction_thresholds_set_soft_memory_available
-    - kubelet_eviction_thresholds_set_soft_nodefs_available
-    - kubelet_eviction_thresholds_set_soft_nodefs_inodesfree
-    - kubelet_eviction_thresholds_set_soft_imagefs_available
-    - kubelet_eviction_thresholds_set_soft_imagefs_inodesfree
     - kubelet_eviction_thresholds_set_hard_memory_available
     - kubelet_eviction_thresholds_set_hard_nodefs_available
     - kubelet_eviction_thresholds_set_hard_nodefs_inodesfree
     - kubelet_eviction_thresholds_set_hard_imagefs_available
-    - kubelet_eviction_thresholds_set_hard_imagefs_inodesfree
```

Comment:
I understand why you removed the `soft` thresholds, but can you remind me why was the check for the `_hard_imagefs_inodesfree` attribute removed?

Reply:
Because `imagefs_inodesFree` is not part of default eviction parameter